plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/data/aggregate/FunctionCountNulls.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2022 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.model.data.aggregate;
+
+public class FunctionCountNulls implements IAggregateFunction {
+    private long count = 0;
+
+    @Override
+    public boolean accumulate(Object value, boolean aggregateAsStrings) {
+        if (value == null) {
```

Comment:
I was thinking a lot and believe you would benefit from using `DBUtils#isNullValue` here.

The reason is that complex values, such as composites and arrays, aren't `null` but their contents is `null`, and that's what the proposed method is looking for.

You don't need to alter test or anything, it will be a drop-in replacement.

Reply:
Done.